content/partner/junzi.md

```diff
@@ -0,0 +1,10 @@
+---
+type: partner
+name: juzi
+kind: company
+displayName: 句子秒回
+logo: juzi.png
+homepage: https://wechat.botorange.com
```

Comment:
Can we change this to `https://www.botorange.com`?

Reply:
Of course!